src/Application/Regulation/View/NumberedRoadView.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Application\Regulation\View;
+
+final readonly class NumberedRoadView implements LocationViewInterface
+{
+    public function __construct(
+        public string $roadType,
```

Comment:
Ici on peut aussi hardcoder ?

    public string $roadType = RoadTypeEnum::DEPARTMENTAL_ROAD->value;

Mais c'est un peu plus bizarre car on parle de NumberedRoadView mais le type s'appelle `DEPARTEMENTAL_ROAD`

Ça me fait dire que cette PR introduit une petite confusion entre "Route numérotée" et "Route départementale", qu'on devrait peut-être résoudre tout de suite ?

Actuellement on confond les deux car on ne gère que la route départementale

Mais expérience de pensée : quand on introduira le type Nationale (par ex), que devra-t-on faire ?

Concrètement le formulaire sera le même entre les deux : gestionnaire, numéro de route, points de repère. La seule diff pourrait être la source de l'autocomplete : 1) restreindre aux Dxx OU BIEN Nxx => on aurait alors un choix de select différent pour Route départementale / Route nationale ? Ou 2) Accepter tout type de route de façon "unifiée" dans le Numéro de route => donc un choix de select unifié comme "Route départementale ou nationale" ?

Je pense que le cas 2) sera le plus simple et surtout le plus en phase avec cette notion de NumberedRoad / Route numérotée. Dans la BD TOPO les routes départementales et nationales (et autoroutes etc) sont bien dans la même table `route_numerotee_ou_nommee`

DONC : est-ce qu'il ne faut pas renommer le `RoadTypeEnum::DEPARTEMENTAL_ROAD` en `RoadTypeEnum::NUMBERED_ROAD` ? Et c'est seulement dans l'UI (les traductions) qu'on continuerait à mettre "Route départementale" (pour l'instant) ?

Reply:
> DONC : est-ce qu'il ne faut pas renommer le RoadTypeEnum::DEPARTEMENTAL_ROAD en RoadTypeEnum::NUMBERED_ROAD ? Et c'est seulement dans l'UI (les traductions) qu'on continuerait à mettre "Route départementale" (pour l'instant) ?

Je suis assez d'accord avec toi, mais je préfèrerai le faire dans une autre PR, celle-ci étant déjà assez grosse et bloquante pour la suite (la #741 notamment)

Reply:
Yes je suis d'accord, je crois avoir ensuite noté dans un commentaire de faire ça dans une autre PR aussi